inference-engine/src/gna_plugin/gna_device.hpp

```diff
@@ -55,7 +55,9 @@ class GNADeviceHelper {
     Gna2DeviceVersion detectedGnaDevVersion = Gna2DeviceVersionSoftwareEmulation;
     std::string executionTarget;
     std::string compileTarget;
+    bool useDeviceEmbedded1_0 = false;
```

Comment:
Rename `useDeviceEmbedded1_0` into more generic `useDeviceEmbedded`
```suggestion
    bool useDeviceEmbeddedExport = false;
```